ext/spl/spl_fixedarray.c

```diff
@@ -188,15 +189,15 @@ static void spl_fixedarray_resize(spl_fixedarray *array, zend_long size)
 	if (size == 0) {
 		spl_fixedarray_dtor(array);
 		array->elements = NULL;
+		array->size = 0;
```

Comment:
Shouldn't this be set before `spl_fixedarray_dtor` to avoid the same issue on deallocation of the whole array?

Reply:
I'll have a closer look, I'm currently fixing the case of calling resize within resize, still causing uaf :/ I think I have a solution for that though.
I'll have a deeper look at your comment after that.